requirements/README.rst

```diff
@@ -12,22 +12,27 @@ directly in the requirements directory.)
 
 .. _OEP-18: https://github.com/openedx/open-edx-proposals/blob/master/oeps/oep-0018-bp-python-dependencies.rst
 
-Upgrading all dependencies
-**************************
+While the ``*.in`` files are intended to be updated manually, the updating the ``*.txt`` files should only be manipulated using Makefile targets in a Linux environment (to match our build and deploy systems). Developers on Mac should prefer to use the GitHub workflows or run Make targets from inside devstack's lms-shell or another Linux environment.
```

Comment:
Note to self: Should tell people to run this on their edx-platform fork if they don't have permissions on the main one.